app/javascript/controllers/replace_controller.js

```diff
@@ -0,0 +1,17 @@
+import { Controller } from "@hotwired/stimulus"
```

Comment:
Maybe creating a `tooltip_controller` is a better abstraction here? The reusable tooltip HTML can also be inside this js file instead of a rails helper.

Also, I don't know if it's worth the work instead of just using a React tailwind/tooltip library.

Reply:
I agree with the tooltip controller, I am going to try change it, if it took too much time, we leave it as it is